ivy/functional/frontends/tensorflow/math.py

```diff
@@ -711,3 +711,8 @@ def real(input, name=None):
 @to_ivy_arrays_and_back
 def atanh(x, name="atanh"):
     return ivy.atanh(x)
+
+
```

Comment:
Hi both, `ivy.gamma` had a few issues regarding dtype handling, which I have now resolved. Please merge the latest master and try running the frontend again. 
Also, kindly add the supported dtypes decorator here and use `available_dtypes=helpers.get_dtypes("valid")` in the test. This is a matter of good practice to make sure all our functions are well-defined. Thanks!

Reply:
Thanks very much!

If you don't mind me asking, what do you mean by the changes to the decorators? I'm assuming switching `get_dtypes("float")` to `get_dtypes("valid")` will refer to the types provided by the supported dtypes decorator or something else trustworthy, but I'm not sure which decorator "supported dtypes" refers to.

Reply:
`@with_supported_dtypes` (or `@with_supported_device_and_dtypes` in case the supported dtypes are different across devices)

Reply:
Thanks very much! I've since added those changes, although I might have fumbled my use of git somewhat.